src/components/SectionTitle/SectionTitle.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Box, BoxProps, makeStyles } from '@material-ui/core';
+
+export type SectionTitleProps = BoxProps;
+
+const useStyles = makeStyles((theme) => ({
+  SectionTitle: {
+    fontSize: '24px',
+    color: theme.palette.text.primary,
+  },
+  divider: {
+    height: '2px',
+    marginTop: '10px',
```

Comment:
It's my mistake on the mockup. Use spacing of 8 or 12px and use `theme.spacing` function for it